### ISSUE TITLE
Error creating Vlans

### ISSUE THREAD

**Issue by danielmenzen** [networklore/ansible-cisco-snmp#2]
First of all, these modules are AWESOME!

I'm trying to figure out how to solve this error:

```
failed: [x.x.x.x] => {"failed": true, "parsed": false}
Traceback (most recent call last):
  File "/home/ubuntu/.ansible/tmp/ansible-tmp-1446051069.8-34119550336394/cisco_snmp_vlan", line >1914, in <module>
    main()
  File "/home/ubuntu/.ansible/tmp/ansible-tmp-1446051069.8-34119550336394/cisco_snmp_vlan", line >1898, in main
    vartable = dev.getnext(o.vtpVlanEditTable)
  File "/usr/local/lib/python2.7/dist-packages/nelsnmp/snmp.py", line 278, in getnext
    lookupMib=False
  File "/usr/local/lib/python2.7/dist-packages/pysnmp/entity/rfc3413/oneliner/cmdgen.py", line 222, in nextCmd
    return errorIndication, errorStatus, errorIndex, varBindTable
 UnboundLocalError: local variable 'errorIndication' referenced before assignment

 FATAL: all hosts have already failed -- aborting
```

It just happens when I try to create a Vlan on the device, I'm testing using a Catalyst 2960. This error occurs for both SNMP v2c and v3.

Everything else works fine!

Thanks

**Comment by ogenstad** [comment]
Hi Daniel,

I'm glad you like the modules and thanks for reporting the issue!

I've done some testing and as far as I can see problem lies with changes introduced in pysnmp version 4.3.0. I've had to do some changes to my code after the people behind pysnmp changed some default behaviour.

I'll have to do some digging don't yet know if this is a bug in pysnmp or if I have to do some changes because of their changes.

As a workaround this should work:

``` bash
pip uninstall pysnmp
pip install pysnmp==4.2.5
```

Please let me know if that works for you and I'll try to solve the problem.

Best regards
Patrick

**Comment by danielmenzen** [comment]
Thank you very much Patrick, it solved the problem!

**Comment by ogenstad** [comment]
Ok good.

This seems to be a bug in pysnmp 4.3.0 and looks to be fixed in pysnmp 4.3.1 which hasn't yet been released.

**Comment by ogenstad** [comment]
Forgot to close this issue. Closing now as pysnmp 4.3.1 has been released.
